### PR TITLE
Remove default homepageUrl functionality and dont show include link i…

### PR DESCRIPTION
**Pull request by LBHELewis**
…f homepageUrl absent

### CODE REVIEW

File: src/components/Header.test.tsx [hunk 1]
```diff
@@ -32,7 +32,6 @@ it("renders correctly without flags", () => {
     <Header
       id="test-id"
       className="test-class"
-      homepageUrl="/test-home"
```
Comment (erbridge) [inline src/components/Header.test.tsx]: I think this still belongs in this test.

File: src/components/Header.tsx [hunk 2]
```diff
@@ -75,6 +75,22 @@ export const Header: React.FunctionComponent<HeaderProps> = (
 
   const Title = isHome ? "h1" : "div";
 
+  const Logo = (
```
Comment (erbridge) [inline src/components/Header.tsx]:
```suggestion
  const logo = (
```

TitleCase implies that it's a class or component, not an instance.

**Comment by LBHELewis** [comment]
@erbridge Updated, thanks

**Review by erbridge — APPROVED** [review]
Looking good.

**Pull request merged**